packages/core/src/application.ts

```diff
@@ -14,13 +14,13 @@ import {Component, mountComponent} from './component';
 import {CoreBindings} from './keys';
 import {
   asLifeCycleObserverBinding,
-  isLifeCycleObserver,
   isLifeCycleObserverClass,
   LifeCycleObserver,
 } from './lifecycle';
 import {Server} from './server';
-import debugModule = require('debug');
-const debug = debugModule('loopback:core:application');
+const CoreTags = CoreBindings.Tags;
```

Comment:
There are quite few  places where we are aliasing `CoreBindings.Tags` to `CoreTags`. I see it as a clear sign that tags should be a new top-level export and not a nested namespace inside `CoreBindings`. I.e.

```ts
import {CoreBindings, CoreTags} from './keys';
```

Thoughts?